doc/source/changes/version_0_29.rst.inc

```diff
@@ -184,6 +184,8 @@ Miscellaneous improvements
 
 * updated documentation of argument 'keepaxes' of aggregation functions (closes :issue:`607`).
 
+* renamed function `element_equal` as `element_equals` (closes :issue:`630`).
```

Comment:
Rule: use s in all function names like in pandas and xarray.

Reply:
This is silly, it does not make sense grammatically to always use s. Is there more example of functions with s in pandas and xarray than just "equals"? AFAIK, pandas does not have an element_equal function in its API.

Reply:
We already have ``Axis.equals``, ``Group.equals``, ``LArray.equals``, ``Session.equals`` and ``Session.element_equals``. So for consistency, I would choose ``element_equals``.
For the justification, let say: 
Test if each element of an array equals the corresponding element of an other array.

Reply:
Ok, but the "justification" text should appear in the docstring. But now that you mention Session.element_equals, I wonder if this should not be a method on LArray instead. And, if we do that, I wonder if it shouldn't be called "eq" like in Pandas instead of "element_equals" because we will need to have methods for all other binary ops in the future (to pass extra arguments), like pandas does. Calling the methods the same as the binop (modulo the underscores) seems like a reasonable thing to me.

Reply:
I was also wondering if `element_equals` shouldn't be a method of LArray. 
About renaming `element_equals` as `eq`, this means we need to rename it also in Session, right? (just to be sure). 

    Calling the methods the same as the binop (modulo the underscores)

underscores?

Reply:
no. If we have LArray.eq (equivalent to current element_equal), we will indeed need a Session.eq at some point too. But that method should be equivalent by default to __eq__ (and in fact could be the same method) and that method returns a Session of LArrays, not a Session of booleans. eq should just offer extra options/keyword arguments over `__eq__`

In other words, currently `Session.__eq__` does the same as calling `__eq__` on each contained object (I think we should keep it like that), but this is **not** the same than Session.element_equals which call object .equals on each contained object. 

In summary, we currently have 3 methods on Session:
* == (`__eq__`): returns Session of LArray
* element_equals: returns a Session of bools
* equals: returns a single bool

An eq method should be equivalent to == (`__eq__`).
element_equals *could* disappear in the (long term?) future when we implement aggregate functions on sessions with a special "element" or "object" axis. When that will be implemented, sess1.element_equals(sess2) will be equivalent to : (sess1 == sess2).all_by('object'). Now I am unsure if at that point we will want to keep element_equals as a shortcut or deprecate it. But until then, we surely need to keep it.